packages/itinerary-body/src/AccessLegBody/index.tsx

```diff
@@ -109,40 +112,57 @@ class AccessLegBody extends Component<Props, State> {
             showLegIcon={showLegIcon}
           />
           <S.LegDetails>
-            <S.StepsHeaderAndMapLink>
-              <S.StepsHeader
-                aria-expanded={expanded}
-                onClick={this.onStepsHeaderClick}
+            {hideDrivingDirections ? (
+              <S.StepsHeaderAndMapLink>
+                <S.StepsHeader as="span" span>
+                  <Duration seconds={leg.duration} />
+                </S.StepsHeader>
+                <MapillaryButton
+                  clickCallback={mapillaryCallback}
+                  coords={leg.from}
+                  mapillaryKey={mapillaryKey}
+                />
+              </S.StepsHeaderAndMapLink>
+            ) : (
+              <S.StepsHeaderAndMapLink>
+                <S.StepsHeader
+                  aria-expanded={expanded}
+                  onClick={this.onStepsHeaderClick}
+                >
+                  <Duration seconds={leg.duration} />
+                  {leg.steps && <S.CaretToggle expanded={expanded} />}
+                  {!hideDrivingDirections && (
```

Comment:
This condition is now redundant, you can remove it.

Reply:
Whoops, good catch thank you! Fixed in [a2374ac](https://github.com/opentripplanner/otp-ui/pull/658/commits/a2374ac0508f42fa387e4659b5fd306722c96f40)